### PR TITLE
Removes black box from Py

### PR DESCRIPTION
**Pull request by rdaly525**

### CODE REVIEW

File: peak/family.py [hunk 1]
```diff
@@ -225,6 +225,8 @@ def get_adt_t(self, adt_t):
     def get_constructor(self, adt_t):
         return adt_t
 
+    def assemble(self, locals, globals, **kwargs):
+        return super().assemble(locals, globals, **kwargs)
```
Comment (cdonovick) [inline peak/family.py:229]: shouldn't need this

**Review by cdonovick — APPROVED** [review]
approved but please address comment

File: peak/family.py (hunk 1 above)
Comment (rdaly525) [reply peak/family.py:229]: I was getting an error because the parent class is abstract

Comment (cdonovick) [reply peak/family.py:229]: Ah I seea

**Pull request merged**